_custom_build/backend.py

```diff
@@ -0,0 +1,28 @@
+import sys
+
+from setuptools.build_meta import *  # noqa: F401, F403
+from setuptools.build_meta import build_wheel
+
+backend_class = build_wheel.__self__.__class__
+
+
+class _CustomBuildMetaBackend(backend_class):
+    def run_setup(self, setup_script="setup.py"):
+        flags = []
+        if (
+            self.config_settings
+            and self.config_settings.get("electrum_ecc.dont_compile") == "true"
+        ):
+            flags.append("electrum_ecc.dont_compile")
```

Comment:
So now instead of 1 thing, there would be 3 :/
- the `electrum_ecc.dont_compile` flag
- the `electrum_ecc.dont_compile` setting that can e.g. take a `true` value
- the `ELECTRUM_ECC_DONT_COMPILE` env var that can take a `1` value, or e.g. a `0` value, or be unset

I would like these to be consolidated. All three should take the same kind of values as the env var: 1 or 0 or be unset. You should just keep passing around the same value, instead of doing conversions.

Reply:
Yeah, I can make it --electrum_ecc.dont_compile=1/0 if you want
which will pass the exactly same thing to setup.py
Will do